sdk/storage/Azure.Storage.Blobs/perf/Azure.Storage.Blobs.Perf/Options/PartitionedTransferOptions.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Storage.Blobs.Perf.Infrastructure.Models.ClientSideEncryption;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Test.Perf;
+using CommandLine;
+
+namespace Azure.Storage.Blobs.Perf.Options
+{
+    public class PartitionedTransferOptions : SizeOptions, IBlobClientOptionsOptions, IStorageTransferOptionsOptions
+    {
+        private int? _maximumTransferLength;
+        private int? _maximumConcurrency;
+
+        [Option('l', "maximumTransferLength")]
+        public int? MaximumTransferLength
+        {
+            get => _maximumTransferLength;
+            set
+            {
+                _maximumTransferLength = value;
+                UpdateStorageTransferOptions();
+            }
+        }
+
+        [Option('t', "MaximumConcurrency")]
+        public int? MaximumConcurrency
+        {
+            get => _maximumConcurrency;
+            set
+            {
+                _maximumConcurrency = value;
+                UpdateStorageTransferOptions();
+            }
+        }
+
+        [Option("clientEncryptionVersion")]
+        public string EncryptionVersionString { get; set; }
```

Comment:
I'd check if the CommandLineParser can handle enums.
See https://github.com/commandlineparser/commandline/wiki/Enum-Flags

Reply:
It seems this is only possible if I provide the string name of the enum value. However, those values in here are `V1_0` and `V2_0`, which are not the values we want to provide (`1.0` and `2.0`). So I have to do the conversion myself it seems.

The actual storage library has internal extension methods that do the conversion.